doc/src/site/sphinx/deployment/rsparkling_on_windows.rst

```diff
@@ -135,6 +135,6 @@ Troubleshooting
 References
 ~~~~~~~~~~
 
-- `Main RSparkling Documentation <../rsparkling.rst>`__
+- `Main RSparkling Documentation <docs.h2o.ai/sparkling-water/master/bleeding-edge/doc/rsparkling.html>`__
```

Comment:
It would be easier to do it using the references in Spinx such as:

Main RSparkling Documentation :ref:`rsparkling_main` and definene the reference as `.. _rsparkling_main:`

This way we don't need to worry about changing the link for master and release branches. Could you please change it?

Reply:
Sure, that makes sense.